bower.json

```diff
@@ -14,7 +14,7 @@
     "svg-piechart": "~1.0.1",
     "akc-meta": "git://github.com/akc42/akc-meta.git#b0a6bde0e",
     "file-upload": "~1.6.1",
-    "material-progress": "^0.1.2"
+    "material-progress": "0.1.2"
```

Comment:
oh, that was left over. It's a seperate issue, I get an error with the progress bars, card-chart:68 gets undefined instead of an array obj, so it doesn't render progress bars